src/jdk.jdwp.agent/share/native/libjdwp/eventHandler.c

```diff
@@ -546,11 +546,6 @@ filterAndHandleEvent(JNIEnv *env, EventInfo *evinfo, EventIndex ei,
         HandlerNode *node;
         char        *classname;
 
-        /* We must keep track of all classes prepared to know what's unloaded */
-        if (evinfo->ei == EI_CLASS_PREPARE) {
-            classTrack_addPreparedClass(env, evinfo->clazz);
-        }
-
```

Comment:
We no longer need this hack here since the class tracking code now installs its own CLASS_PREPARE event handler.